data/data/alibabacloud/bootstrap/outputs.tf

```diff
@@ -0,0 +1,27 @@
+output "bucket_id" {
+  value = alicloud_oss_bucket.bucket.id
+}
+
+output "bootstrap_role_name" {
+  value = alicloud_ram_role.role.name
+}
+
+output "bootstrap_role_arn" {
+  value = alicloud_ram_role.role.arn
+}
+
+output "bootstrap_role_policy_id" {
+  value = alicloud_ram_policy.role_policy.id
+}
+
+output "bootstrap_sg_id" {
+  value = alicloud_security_group.sg_bootstrap.id
+}
+
+output "bootstrap_ecs_id" {
+  value = alicloud_instance.bootstrap.id
+}
+
+output "bootstrap_ecs_ip" {
+  value = data.alicloud_instances.bootstrap_data.instances.0.private_ip
+}
```

Comment:
These don't appear to be being used. Any resources allocated in the bootstrap stage that aren't permanent should not be passed to the next stage. This should also output the bootstrap IP.

Reply:
OK, I only kept the bootstrap IP. thx